static/app/views/settings/project/projectKeys/details/loaderSettings.tsx

```diff
@@ -226,8 +226,12 @@ export function LoaderSettings({keyId, orgSlug, project, data, updateData}: Prop
                 ? t('Only available in SDK version 7.x and above')
                 : data.dynamicSdkLoaderOptions.hasReplay
                   ? tct(
-                      'When using Replay, the loader will load the ES6 bundle instead of the ES5 bundle. The default configurations are [codeReplay:replaysSessionSampleRate: 0.1] and [codeError:replaysOnErrorSampleRate: 1]. [configDocs:Read the docs] to learn how to configure this.',
+                      `[es5Warning]The default configurations are [codeReplay:replaysSessionSampleRate: 0.1] and [codeError:replaysOnErrorSampleRate: 1]. [configDocs:Read the docs] to learn how to configure this.`,
                       {
+                        es5Warning:
+                          data.browserSdkVersion !== '8.x'
```

Comment:
Wondering if we should flip it to `=== 7.x` cause that will be more correct in the future. I am wondering in general when we can remove this warning.

Reply:
I guess we can do that 👍